cypress.config.js

```diff
@@ -18,4 +18,5 @@ module.exports = defineConfig({
       return config;
     },
   },
+  retries: 2,
```

Comment:
I see this was introduced earlier already on master. But I believe this is a general anti-pattern when dealing with flaky tests. I strongly discourage from using any retries since we will have even less insights into why our tests or our UI is flaky.

Reply:
In the end this usually didn't help with anything, if it failed once then it failed in the retries as well. But I agree it shouldn't be there